[2025-02-22T16:51:50.167+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-22T16:51:50.194+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: london_weather_pipeline.extract_weather_data manual__2025-02-22T16:51:49.397463+00:00 [queued]>
[2025-02-22T16:51:50.202+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: london_weather_pipeline.extract_weather_data manual__2025-02-22T16:51:49.397463+00:00 [queued]>
[2025-02-22T16:51:50.203+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-22T16:51:50.221+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): extract_weather_data> on 2025-02-22 16:51:49.397463+00:00
[2025-02-22T16:51:50.227+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3178) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-22T16:51:50.228+0000] {standard_task_runner.py:72} INFO - Started process 3179 to run task
[2025-02-22T16:51:50.229+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'london_weather_pipeline', 'extract_weather_data', 'manual__2025-02-22T16:51:49.397463+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/london_weather_pipeline/london_weather_etl.py', '--cfg-path', '/tmp/tmp6t1i8p7f']
[2025-02-22T16:51:50.230+0000] {standard_task_runner.py:105} INFO - Job 24: Subtask extract_weather_data
[2025-02-22T16:51:50.297+0000] {task_command.py:467} INFO - Running <TaskInstance: london_weather_pipeline.extract_weather_data manual__2025-02-22T16:51:49.397463+00:00 [running]> on host ee2081ae3a04
[2025-02-22T16:51:50.410+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='london_weather_pipeline' AIRFLOW_CTX_TASK_ID='extract_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T16:51:49.397463+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-22T16:51:49.397463+00:00'
[2025-02-22T16:51:50.410+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-22T16:51:50.429+0000] {base.py:84} INFO - Retrieving connection 'open_meteo_id'
[2025-02-22T16:51:51.001+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': 51.5, 'longitude': -0.120000124, 'generationtime_ms': 0.03981590270996094, 'utc_offset_seconds': 0, 'timezone': 'GMT', 'timezone_abbreviation': 'GMT', 'elevation': 16.0, 'current_weather_units': {'time': 'iso8601', 'interval': 'seconds', 'temperature': '°C', 'windspeed': 'km/h', 'winddirection': '°', 'is_day': '', 'weathercode': 'wmo code'}, 'current_weather': {'time': '2025-02-22T16:45', 'interval': 900, 'temperature': 12.9, 'windspeed': 13.7, 'winddirection': 252, 'is_day': 1, 'weathercode': 2}}
[2025-02-22T16:51:51.037+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-22T16:51:51.037+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=london_weather_pipeline, task_id=extract_weather_data, run_id=manual__2025-02-22T16:51:49.397463+00:00, execution_date=20250222T165149, start_date=20250222T165150, end_date=20250222T165151
[2025-02-22T16:51:51.084+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-22T16:51:51.161+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-22T16:51:51.165+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
